bin/population/pycbc_multiifo_pastro

```diff
@@ -0,0 +1,396 @@
+parser.add_argument('--bg-bin-width', default=None,
+                    help='Width of stat bins to estimate background density, eg 0.25')
+parser.add_argument('--inj-bin-width', default=None,
+                    help='Width of bins to estimate injection density, eg 0.4')
+parser.add_argument('--bg-log-bin-width', type=float, default=None,
+                    help='Width of log spaced bins to estimate background'
+                         ' density, eg 0.02')
+parser.add_argument('--inj-log-bin-width', type=float, default=None,
+                    help='Width of log spaced bins to estimate injection'
+                         ' density, eg 0.04')
+parser.add_argument('--min-mchirp', type=float, required=True,
+                    help='Minimum template chirp mass to accept')
+parser.add_argument('--max-mchirp', type=float, required=True,
+                    help='Maximum template chirp mass to accept')
+parser.add_argument('--power-law-prior', default=-0.5, type=float,
+                    help='Power-law prior on foreground count parameter, '
+                         'default -0.5 ("Jeffreys")')
+parser.add_argument('--laguerre-degree', type=int, default=20,
+                    help='Degree of polynomial used for generalized L-G quadrature')
+parser.add_argument('--ntop', type=int, default=15,
+                    help='Number of loudest events listed, default 15')
+parser.add_argument('--p-astro-txt', help='Save GPS and p_astro in txt file')
+parser.add_argument('--p-astro-inj', help='Save injection GPS / p_astro in txt file.'
+                    ' Optional')
+parser.add_argument('--json-tag', help='If given, save event json files with tag '
+                    'in name')
+parser.add_argument('--json-min-ifar', type=float, help='IFAR threshold to save '
+                    'json files')
+parser.add_argument('--vt-mean', type=float,
+                    help='Sensitivity estimate conjugate to merger rate: if '
+                         'provided, rate estimate will be output in the same units')
+parser.add_argument('--vt-err', type=float,
+                    help='Standard error on vt-mean value, assumed in same units')
+parser.add_argument('--rate-units', default=r'Gpc^-3 yr^-1',
+                    help=r'Sensitivity units for rate output, default 1/(Gpc^3 yr)')
+parser.add_argument('--plot-max-stat', type=float, default=30.,
+                    help='Maximum background stat value to plot')
+parser.add_argument('--plot-bg-hist', action='store_true',
+                    help='Make diagnostic background PDF plots')
+parser.add_argument('--plot-inj-hist', action='store_true',
+                    help='Make diagnostic injection PDF plots')
+parser.add_argument('--plot-dir', required=True, help='Destination for plots, '
+                    'must include final "/"')
+parser.add_argument('--plot-tag', help='String for plot filenames')
+
+args = parser.parse_args()
+
+assert args.plot_dir.endswith('/')
```

Comment:
it would be better to either dynamically add this (i.e. if not endswith('/'), += '/'), or use os.path.join

Reply:
also an `os.path.exists(arg.plot_dir)` check would be useful to fail here straight away if the output directory doesn't exists